tools/bowtie2.wdl

```diff
@@ -0,0 +1,410 @@
+        ref_gap_open_extend: "Sets the reference gap open and extend penalties. A reference gap of length N gets a penalty of <int1> + N * <int2>."
+        max_aln_report: {
+            description: "Report up to N alignments per read; MAPQ not meaningful",
+            help: "When specified, bowtie2 searches for at most N distinct, valid alignments for each read. The search terminates when it can't find more distinct valid alignments, or when it finds <int>, whichever happens first. All alignments found are reported in descending order by alignment score. The alignment score for a paired-end alignment equals the sum of the alignment scores of the individual mates. Each reported read or pair alignment beyond the first has the SAM 'secondary' bit (which equals 256) set in its FLAGS field. For reads that have more than N distinct, valid alignments, bowtie2 does not guarantee that the N alignments reported are the best possible in terms of alignment score.",
+        }
+        report_all_alignments: "Report all alignments; very slow, MAPQ not meaningful"
+        min_fragment_len: {
+            description: "The minimum fragment length for valid paired-end alignments.",
+            bowtie2_option: "I|minins",
+        }
+        max_fragment_len: {
+            description: "The maximum fragment length for valid paired-end alignments.",
+            bowtie2_option: "X|maxins",
+        }
+        no_mixed: "By default, when bowtie2 cannot find a concordant or discordant alignment for a pair, it then tries to find alignments for the individual mates. This option disables that behavior."
+        no_discordant: "By default, bowtie2 looks for discordant alignments if it cannot find any concordant alignments. A discordant alignment is an alignment where both mates align uniquely, but that does not satisfy the paired-end constraints. This option disables that behavior."
+        dovetail: {
+            description: "If the mates 'dovetail', that is if one mate alignment extends past the beginning of the other such that the wrong mate begins upstream, consider that to be concordant.",
+            external_help: "https://bowtie-bio.sourceforge.net/bowtie2/manual.shtml#mates-can-overlap-contain-or-dovetail-each-other",
+        }
+        no_contain: {
+            description: "If one mate alignment contains the other, consider that to be non-concordant.",
+            external_help: "https://bowtie-bio.sourceforge.net/bowtie2/manual.shtml#mates-can-overlap-contain-or-dovetail-each-other",
+        }
+        no_overlap: {
+            description: "If one mate alignment overlaps the other at all, consider that to be non-concordant.",
+            external_help: "https://bowtie-bio.sourceforge.net/bowtie2/manual.shtml#mates-can-overlap-contain-or-dovetail-each-other",
+        }
+        time: "Print the wall-clock time required to load the index files and align the reads to STDERR."
```

Comment:
Is this bloat? I think it would be fine to hardcode this one way or the other. I don't see a use in being able to toggle.